src/components/ProfileInfo/ProfileInfo.tsx

```diff
@@ -41,6 +63,14 @@ const ProfileInfo: React.FC<ProfileInfoProps> = ({ user }) => {
             <span className={`${classes['c-profile-info__number']}`}>{postsNumber}</span>
             <span className={`${classes['c-profile-info__label']}`}>Posts</span>
           </div>
+
+          {user.id === currentLoggedUser.id ? (
+            <></>
+          ) : friends?.some((friend) => friend.id === currentLoggedUser.id) ? (
+            <Button label='Friends' disabled />
+          ) : (
+            <Button label='Send request' onClick={handleSendRequest} />
+          )}
```

Comment:
This is not how we want to do conditional rendering. You can move this to function and render what you need